### ISSUE TITLE
Argument to checks if the documentation is up-to-date

### ISSUE THREAD

**Issue by vzabawski** [norwoodj/helm-docs#143]
It would be nice to have an argument which allows running helm-docs for checking if README is up-to-date.
Such argument is very handy in CI workflows when you just need to check if the documentation is relevant, not to update it.

Here's how it might look like:

Running helm-docs in repo with up-to-date readme
```
$ helm-docs --check
Up-to-date
$ echo $?
0
```

Running helm-docs in repo with outdated readme
```
$ helm-docs --check
README needs to be updated
$ echo $?
2
```

**Comment by GMartinez-Sisti** [comment]
I would also like to see this. We're using git to check if anything changed, but ideally the exit code would work for this.

```yaml
      - name: Ensure documentation is updated
        run: |
          docker run --rm --volume \
            "$(pwd):/helm-docs" \
            -u "$(id -u)" \
            jnorwood/helm-docs:v1.11.0

          if git diff --exit-code; then
            echo -e "\n####### Git is clean\n"
          else
            echo -e "\n####### Git changes detected! Check and commit changes !!!\n"
            exit 1
          fi
```

**Comment by Nepo26** [comment]
Might be a good thing to check for it. Will add on the list to look further into it.

**Comment by jhoover4** [comment]
Do you need any help with this @Nepo26? I'd really like to have this and I'd be happy to pitch in if you need the help

**Comment by Nepo26** [comment]
Yes, I would like some help @jhoover4. Can you check if this falls under the same function of strict mode? If not, can you provide some examples of how do you think this function should work? I don't think a `README needs to be updated` is enough, this probably can be more verbose.

**Comment by jhoover4** [comment]
Yeah I'm honestly not sure if we need this after we include strict mode. If strict mode is on and you dont want an aspect documented, you should just need to put `@ignore` right? I think we should focus more on making sure all the corner cases for strict mode are good.
